frontend/src/components/Intervention.jsx

```diff
@@ -9,25 +9,23 @@ const { apiSite } = require("../conf");
 function Intervention(props) {
   const [nextMaintenance, setnextMaintenance] = useState([]);
   const [passedMaintenance, setpassedMaintenance] = useState([]);
+  const token = useSelector(state => state.user.token);
 
   useEffect(() => {
-    axios.get(`${apiSite}/vehicule/1/nextmaintenance`).then(({ data }) => {
-      const lvls = calcLevels(data);
-
-      setnextMaintenance(
-        lvls
-          .filter(vehicule => {
+    axios
+      .get(`${apiSite}/vehicule/1/nextmaintenance`, {
```

Comment:
This `1` is gonna come back and bite you in the ass

Reply:
it's a bugfix branch, this kind of problem will be resolve in another branch